api/v2/views/machine_request.py

```diff
@@ -27,6 +30,20 @@ class MachineRequestViewSet(BaseRequestViewSet):
     ordering_fields = ('start_date', 'end_date', 'new_machine_owner__username')
     ordering = ('-start_date',)
 
+    def get_queryset(self):
+        if 'active' in self.request.query_params:
+            all_active = MachineRequest.objects.filter(
+                (
+                    Q(status__name='pending') |
```

Comment:
Can we change this to: `status__name_ne='closed'`? this way we can avoid 'losing track of' tickets that are open and in a state other than pending.